BraintreeDropIn/BTCardFormViewController.m

```diff
@@ -826,6 +837,16 @@ - (void)formFieldDidChange:(BTUIKFormField *)formField {
     }
 }
 
+- (BOOL)formFieldShouldReturn:(BTUIKFormField *)formField {
+    if (formField == self.cardholderNameField) {
+        [self advanceFocusFromField:formField];
+        return NO;
+    }
+    else {
```

Comment:
This `else` case isn't required, we could just `return YES;` if we didn't fall into the `if`